src/components/connectors/bc-alby-nwc-connector.ts

```diff
@@ -16,8 +17,9 @@ export class AlbyNWCConnector extends ConnectorElement {
 
   protected async _onClick() {
     const nwc = webln.NostrWebLNProvider.withNewSecret();
-
+    const providerConfig = store.getState().providerConfig;
     await nwc.initNWC({
+      ...(providerConfig?.nwc?.authorizationUrlOptions || {}),
```

Comment:
does this mean we need to define all the potential provider config options here in BC?

Reply:
no, they're all optional and it just defaults to an empty object